syntax = "proto3";

import "google/protobuf/timestamp.proto";

message Orders_proto {
    int32 id = 1;
    int32 product_id = 2;
    int32 user_id = 3;
    int32 quantity = 4;
    float price = 5;
    string status = 6;
    float total_price = 7;
    google.protobuf.Timestamp created_at = 8;
    google.protobuf.Timestamp updated_at = 9;
}
